crates/router/src/consts.rs

```diff
@@ -91,3 +91,6 @@ pub const MAX_SESSION_EXPIRY: u32 = 7890000;
 pub const MIN_SESSION_EXPIRY: u32 = 60;
 
 pub const LOCKER_HEALTH_CALL_PATH: &str = "/health";
+
+// URL for checking the outgoing call
+pub const OUTGOING_CALL_URL: &str = "https://www.google.com";
```

Comment:
There is a inherent issue here. What if the infra admin has a blocklist, which blocks https://www.google.com/ but has whitelisted the connector endpoints, then this will fail right?

Reply:
But what if that particular connector is down, we can't just say outgoing call is not working right? That's why I used google.com